js/views/message.js

```diff
@@ -167,8 +231,6 @@ views.Messages = Backbone.Marionette.CompositeView.extend({
 	currentMessageId: null,
 
 	events: {
-		"click #load-new-mail-messages" : "loadNew",
-		"click #load-more-mail-messages" : "loadMore"
 	},
```

Comment:
Same as above, latest state of your branch: https://github.com/owncloud/mail/pull/834/files#diff-df51d65ab371ac49786ad664ec2d53dfL169